breathecode/admissions/tests/urls/tests_academy_cohort_user_profile.py

```diff
@@ -0,0 +1,111 @@
+"""
```

Comment:
Bad test name, this need to have any relation with the path of endpoint or the name of it